packages/repository/test/fixtures/models/order.model.ts

```diff
@@ -27,5 +27,6 @@ export class Order extends Entity {
   isShipped: boolean;
 
   @belongsTo(() => Customer)
+  @property()
```

Comment:
IMO, this is a breaking change, since you are requiring all existing users of `belongsTo` to add `@property` decorator to their models. Making a breaking change comes with (maintenance) costs, see https://github.com/strongloop/loopback-next/blob/8394c74d5369d7dcce861c0ed30e033a3e7ce3ae/docs/site/DEVELOPING.md#making-breaking-changes

Reply:
Good point. After reading the guide about frequency of breaking changes and what warrants them/support model, I have decided to revert these changes for now and just keep this PR about usage of unique indexes. @raymondfeng thoughts?